src/cli/templates/create/_._eslintrc

```diff
@@ -0,0 +1,104 @@
+{
+  "parser": "babel-eslint",
+  "extends": "eslint:recommended",
```

Comment:
What do you think about promoting prettier here? You can check our own https://github.com/botpress/botpress/blob/develop/x/.eslintrc for how it can be configured. That way we don't need to define the majority of stylistic rules ourselves

Reply:
Yes, I like to have as little configuration as possible! After making this PR I have started using `standard` personally (https://standardjs.com/).

I like to use it enforced with `lint-staged` and `husky`, like this: https://github.com/hugojosefson/nodejs-example-cli/blob/e9d0a75552e31b79db7ba5d8fcd2de4d7f490eaf/package.json#L23-L30